src/GitLink/GitLink.csproj

```diff
@@ -13,14 +12,12 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\output\debug\GitLink\</OutputPath>
+    <OutputPath>bin\debug\</OutputPath>
```

Comment:
I agree. I really like the "read only source tree" pattern. Which is why I wrote the ReadOnlySourceTree nuget package, which takes care of it automatically. This change reverts the project back to its "fresh from template" state to keep it simple, but it still writes to a top-level bin folder. It also takes care of the obj folder and the .xml file automatically.